jib-core/src/main/java/com/google/cloud/tools/jib/image/json/BuildableManifestTemplate.java

```diff
@@ -43,7 +43,10 @@
   @VisibleForTesting
   class ContentDescriptorTemplate implements JsonTemplate {
 
-    @Nullable private String mediaType;
+    @SuppressWarnings("unused")
```

Comment:
I wonder if it is possible to configure errorpone to exclude classes of type `JsonTemplate`. But I'm not sure which one is better in the end even if it's possible

Reply:
Hrmm.. I'm thinking being more explicit about it means we did it intentionally, rather than wholesale.